WcaOnRails/app/views/contacts/website.html.erb

```diff
@@ -1,7 +1,9 @@
 <% provide(:title, t('.title')) %>
 <div class="container">
   <h1><%= yield(:title) %></h1>
-
+  <div class="alert alert-info">
+      Before making an inquiry, you may wish to take a look at our <%= link_to 'Frequently Asked Questions', '/faq' %>! If you wish to contact a specific committee or team, you can find their contact information on our <%= link_to 'about page', '/about' %>.
+  </div>
```

Comment:
I think in this case you can just hard code the link as an HTML tag, like here:
https://github.com/thewca/worldcubeassociation.org/blob/a78a9f119785c518c449861e77f2906040b71631/WcaOnRails/config/locales/en.yml#L1638
This is generally fine for short links that are constant (i.e. doesn't require any ID).

When doing this, just make sure to suffix the i18n key with `_html` (e.g. `faq_note_html`), so that it gets rendered as HTML, not escaped as a normal text.